### PR TITLE
Ticketing solve misc bugs #2965

### LINKED ISSUE #2965: Miscellaneous bugs in ticketing section
- Payment currency is hard-coded at some places
- Label and ticket holder box overlap
selection_078

### PR DESCRIPTION
**Pull request by shubham-padia**
fixes #2965 .

### CODE REVIEW

File: app/templates/gentelella/admin/mytickets/mytickets_list.html [hunk 1]
```diff
@@ -26,7 +26,7 @@ <h4 class="name">{{ order.event.name }}</h4>
                     <hr>
                     <p class="tags">
                     <span class="filter-hashtag">
-                        {% if order.amount == 0 %}Free{% else %}${{ order.amount | money }}{% endif %}
+                        {% if order.amount == 0 %}Free{% else %} {% if order.event.payment_currency %} {{ order.event.payment_currency | currency_symbol }} {% else %} $ {% endif %} {{ order.amount | money }}{% endif %}
```
Comment (SaptakS) [inline app/templates/gentelella/admin/mytickets/mytickets_list.html]: The `payment_currency` in an event is a required field and is by default set to USD. So I think this `if` check is not necessary.

File: app/templates/gentelella/guest/ticketing/components/_order_summary.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div id="order-summary-holder">
-    <div class="panel panel-default">
+    <div class="panel panel-default" style="margin-top: 20px">
```
Comment (SaptakS) [inline app/templates/gentelella/guest/ticketing/components/_order_summary.html]: Adding `margin-top: 20px` breaks the design in pre_payment page.
screenshot from 2017-01-18 09-44-59

The 2 boxes side-by-side should begin from the same level. 

Firstly the overlap was made consciously. Secondly, even if I consider that as a bug, this doesn't fix the bug.

Comment (shubham-padia) [inline app/templates/gentelella/guest/ticketing/components/_order_summary.html]: :+1:

File: app/templates/gentelella/admin/mytickets/mytickets_list.html (hunk 1 above)
Comment (shubham-padia) [inline app/templates/gentelella/admin/mytickets/mytickets_list.html]: @SaptakS done :)

**Pull request merged**